mdsuite/utils/meta_functions.py

```diff
@@ -444,7 +444,7 @@ def golden_section_search(
         )
 
 
-def round_down(a: int, b: int) -> int:
+def get_nearest_divisor(a: int, b: int) -> int:
```

Comment:
You renamed it here but I don't see the method used in the actual code?

Reply:
Looks like a legacy method. Will remove it.